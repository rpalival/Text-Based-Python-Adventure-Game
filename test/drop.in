go north
get 3d maneuver gear
go south
go east
go north
inventory
drop 3d maneuver gear
look
quit