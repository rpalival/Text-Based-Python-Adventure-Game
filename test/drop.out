> Wall Maria Entrance

The colossal gate of Wall Maria, towering above you. The air is tense with the threat of Titans.

Items: flare gun

Exits: north east

What would you like to do? You go north.

> Scout Regiment HQ

The bustling headquarters of the Scout Regiment. Maps and plans of Titan territory cover the walls.

Items: 3d maneuver gear

Exits: south east

What would you like to do? You pick up the 3d maneuver gear.
What would you like to do? You go south.

> Wall Maria Entrance

The colossal gate of Wall Maria, towering above you. The air is tense with the threat of Titans.

Items: flare gun

Exits: north east

What would you like to do? You go east.

> Shiganshina District

The ruined district where it all began. Abandoned houses and the remnants of a battle linger.

Exits: north west

What would you like to do? You go north.

> Titan Forest

A dense forest, known for its massive trees. A perfect place for ambushes... or to be ambushed.

Exits: east south

What would you like to do? Inventory:
  3d maneuver gear
What would you like to do? You drop the 3d maneuver gear.
What would you like to do? > Titan Forest

A dense forest, known for its massive trees. A perfect place for ambushes... or to be ambushed.

Items: 3d maneuver gear

Exits: east south

What would you like to do? Goodbye!
